src/mmw/js/src/modeling/models.js

```diff
@@ -744,8 +745,20 @@ var ScenarioModel = Backbone.Model.extend({
                 fetchGisDataPromise = App.currentProject.fetchGisDataIfNeeded();
 
             self.fetchResultsPromise = fetchGisDataPromise.then(function() {
-                var promises = fetchResults();
-                return $.when(promises.startPromise, promises.pollingPromise);
+                // Ensure the model completes without modifications before trying to
```

Comment:
By the same logic, is the `gisData` from `fetchGisDataIfNeeded` required for mapshed? Should  `self.fetchResultsPromise = fetchGisDataPromise.then(function() { ...` also be a `.done`?